templates/base.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="`UTF-8`">
```

Comment:
The meta charset tag has backticks around `UTF-8`, which is incorrect. It should be `<meta charset="UTF-8">`.

Reply:
Done